sum2.js

```diff
@@ -0,0 +1,4 @@
+function sum(a, b) {
```

Comment:
@janicelam21 

In general, if this file's intent is to test Jest functionality, I'd typically add it to .gitignore, and remove that line from .gitignore after deleting that file in favor of official tests.

On another note--I love the use of a minimum viable test & ensuring this is setup!

Reply:
Cool - I added it to .gitignore!